qemu/tests/cfg/ple_test.cfg

```diff
@@ -0,0 +1,17 @@
+- ple_test:
+    virt_test_type = qemu
+    type = ple_test
+    not_preprocess = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
```

Comment:
Can we use the default value from VT and delete this line?

Reply:
Sure.